queueWithStacks.py

```diff
@@ -0,0 +1,82 @@
+# Time Complexity : O(N) for push O(1) for pop,peek, empty
+# Space Complexity : O(N) for stackSize
+# Did this code successfully run on Leetcode : Yes
+# Any problem you faced while coding this : Yes. I had a bug within my push function. I needed an else after 
+# the if because I noticed I was appending the first value twice to the stack while I was debugging.
+# another problem I had was solving the wrong problem...I stuck because I thought I was popping from the back 
+# instead of the front of the queue
+
```

Comment:
Good that you are able to debug the code with the logic. Try to write the pseudo code before starting to code. Maybe you'll overcome these mistakes again.